lib/govspeak/presenters/image_presenter.rb

```diff
@@ -0,0 +1,25 @@
+module Govspeak
+  class ImagePresenter
+    attr_reader :image
+
+    def initialize(image)
+      @image = image
+    end
+
+    def url
+      image.respond_to?(:url) ? image.url : image[:url]
+    end
+
+    def alt_text
+      image.respond_to?(:alt_text) ? image.alt_text : image[:alt_text]
```

Comment:
Do we need to do any of the new line trimming that's done in AttachmentImagePresenter? I don't know off the top of my head what issue they were resolving there.

Reply:
I don't think so: partly as we're instead copying Whitehall which seems not to bother (although it might do it in the app); and partly as we know we've got a plain input.

Reply:
ok cool